README.md

```diff
@@ -1,2 +1,48 @@
-# PairStormIDE
-Pair Storm IDE. The project, created by group Lv-425.C++ of the Softserve IT Academy.
+
+
+:cloud: Pair Storm IDE :cloud:
+==============================
+
+[![Code Style](https://img.shields.io/badge/Code%20Style-Qt%20Coding%20Style-green)](https://wiki.qt.io/Qt_Coding_Style) 
+![Contributions](https://img.shields.io/badge/Contributions-Welcome-brightgreen)
+[![License](https://img.shields.io/badge/License-MIT-lightgrey)](/LICENSE) 
+[![Slack](https://img.shields.io/badge/Slack-Join-red)](http://cppteamsoftserve.slack.com/)
+
+
+<img src="src/img/SPLASHSCREEN.png" alt="PairStorm" align="center">
+
+Pair Storm is a project oriented integrated development environment, which based on conception of pair programming. The software contains many useful features, which aim to help programmers to write good and clearly code. About them you can read below. Also Pair Storm helps to organize coworking between two or more programmer. To reaching these goal in the program were added many modules, which help to communicate, share code etc. IDE was created by group LV-425.C++ of the SoftServe IT Academy. 
```

Comment:
clear code

Reply:
You can read about them below

Reply:
two or more programmer**S**

Reply:
Pair Storm is a project oriented IDE that is based on the conception of pair programming. The application contains a bunch of useful features that are going to help programmers write clear and high-quality code. More information about these below. Additionally, Pair Storm supply developers with different sharing features to maintain their team work even remotely. To embed these possibilities, our team has developed custom flexible modular structure, hence Pair Storm is updates-friendly. Created by Lv-425.C++ team under guidance of the SoftServe IT Academy.